.github/workflows/_s3_publish.yml

```
@@ -0,0 +1,282 @@
+              if [[ "${{ steps.version.outputs.release_type }}" == "release" ]]; then
+                S3_PREFIX="stable"
+              else
+                S3_PREFIX="pre"
+              fi
+              ;;
+            *)
+              echo "Unknown target type: ${{ inputs.target_type }}"
+              exit 1
+              ;;
+          esac
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)